practice-one/src/Components/Section/Footer.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FooterLogo = ({ logoSrc, altText }) => (
+  <div className="card-header">
+    <a href="/" className="logo">
+      <img src={logoSrc} alt={altText} />
+    </a>
+  </div>
+);
+
+FooterLogo.propTypes = {
+  logoSrc: PropTypes.string.isRequired,
+  altText: PropTypes.string.isRequired,
+};
+
+const FooterCard = ({ children }) => <div className="card w-[30%] lg:w-[20%]">{children}</div>;
+
+FooterCard.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const FooterCardBody = ({ text }) => (
+  <div className="card-body pt-5 pb-5">
+    <p className="text-sm leading-medium">{text}</p>
+  </div>
+);
+
+FooterCardBody.propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+const FooterMenu = ({ title, children }) => (
+  <div className="footer-menu nav-menu">
+    <ul className="menu-list text-sm">
+      <a href="/" className="item font-bold hover:text-primary">
+        {title}
+      </a>
+      {children}
+    </ul>
+  </div>
+);
+
+FooterMenu.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+const FooterMenuItem = ({ text }) => (
+  <li className="list-item pt-4">
+    <a className="item hover:text-secondary focus:tex-pri" href="/">
+      {text}
+    </a>
+  </li>
+);
+
+FooterMenuItem.propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+const Footer = () => (
+  <footer className="footer bg-white relative mt-56 lg:mt-0 m-7">
+    <div className="footer-content container mx-auto mt-14 mb-12 flex flex-col lg:flex-row flex-wrap justify-between gap-y-8 lg:flex-nowrap">
+      <FooterCard>
+        <FooterLogo logoSrc="src\assets\logo.png" altText="logo" />
+        <FooterCardBody text="We built an elegant solution. Our team created a fully integrated sales and marketing solution for SMBs" />
+        <div className="card-footer">
+          <ul className="media menu-list flex gap-5">
+            <li>

```

Comment:
why dont we reuse it 
![image](https://github.com/Agility-Internship/Duong-The-Hien-ReactJS/assets/27277174/9dc65c7e-5042-4f0f-8652-97702e69501e)

Reply:
Yes, resues FooterMenuItem component and using map to render list